apps/website/foundation/themes/README.md

```diff
@@ -226,7 +207,7 @@ Next, we need to include the the application scss variable overrides and the Cla
 @import "../node_modules/@clr/ui/src/utils/components.clarity";
 ```
 
-##### Why is SASS-based theming going away?
+##### Why are SASS-based theming going away?
```

Comment:
Unfortunately `Why is...` is grammatically correct. Can you revert this change?

Reply:
Reverted. Good catch.